autoblocks/_impl/testing/models.py

```diff
@@ -156,3 +156,40 @@ class BaseEvaluator(
     """
     An ABC for users that are implementing an evaluator that will be run against both test cases and production events.
     """
+
+
+@dataclasses.dataclass
+class ScoreChoice:
+    """
+    A choice used in an LLM judge evaluator.
+    """
+
+    value: float
+    name: str
+
+
+@dataclasses.dataclass
+class EvaluatorOverrideField:
+    id: str
+    name: str
+    value: str
+
+
+@dataclasses.dataclass
+class EvaluatorOverrideComment:
+    field_id: str
+    quoted_text: str
+    comment_text: str
+
+
+@dataclasses.dataclass
+class EvaluatorOverride:
```

Comment:
i think these should all be called Evaluation* overrides, since users are overriding evaluations, not evaluators

Reply:
ya that makes sense, i went back and forth on that